### PR TITLE
Change git user.name and user.email

### PR DESCRIPTION
**Pull request by sinsukehlab**
### Summary

Change git user.name and user.email to display github-actions[bot] as a contributor.

### Changes

.github/workflows/0-welcome.yml
- git user.name: github-actions &rarr; github-actions[bot]
- git user.email: github-actions@github.com &rarr; github-actions[bot]@users.noreply.github.com

### Task list

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by Coste5435 — APPROVED** [review]

File: .github/workflows/0-welcome.yml [hunk 1]
```diff
@@ -74,8 +74,8 @@ jobs:
           printf "%s\n%s\n%s\n\n" "---" "title: Welcome to my blog" "---" > index.md
 
           echo "Make a commit"
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name github-actions[bot]
+          git config user.email github-actions[bot]@users.noreply.github.com
```
Comment (e1b2i3) [inline .github/workflows/0-welcome.yml:78]: Good job

**Review by e1b2i3 — APPROVED** [review]

**Comment by e1b2i3** [comment]
🙏

File: .github/workflows/0-welcome.yml (hunk 1 above)
Comment (e1b2i3) [reply .github/workflows/0-welcome.yml:78]: 🙏

Comment (e1b2i3) [inline .github/workflows/0-welcome.yml:78]: Ebrahim1367@sinsukehlab

**Review by e1b2i3 — CHANGES_REQUESTED** [review]

**Comment by mtyalkwyt** [comment]
9f8e2ea5c9f681ee75081f9c79ff99fa5707ba5d

**Review by Mwipaji — COMMENTED** [review]
SkyMan-Ac-Bot

**Comment by luanxx5** [comment]
收到



发自我的iPhone


------------------ Original ------------------
From: Mwipaji ***@***.***&gt;
Date: Tue,May 27,2025 3:09 PM
To: skills/github-pages ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [skills/github-pages] Change git user.name and user.email (PR#99)





 
@Mwipaji commented on this pull request.
 
SkyMan-Ac-Bot

**Pull request merged**